### ISSUE TITLE
Build issue

### ISSUE THREAD

**Issue by plfiorini** [lirios/materialdecoration#4]
Builds fine with Qt 5.11.1, but with Qt 5.11.2:

```
Building for configuration default
moc materialdecoration.h [materialdecorationplugin]
moc plugin.cpp [materialdecorationplugin]
compiling plugin.cpp [materialdecorationplugin]
compiling moc_materialdecoration.cpp [materialdecorationplugin]
ERROR: /usr/bin/g++ -g -O0 -Wall -Wextra -m32 -pipe -fexceptions -fvisibility=hidden -fvisibility-inlines-hidden -fPIC -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_SERVICE_SUPPORT_LIB -DQT_WAYLANDCLIENT_LIB -I/builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/src/plugins/decorations/material -I/usr/include/qt5 -I/usr/include/qt5/QtCore -I/usr/lib/qt5/mkspecs/linux-g++ -I/builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/build/default/materialdecorationplugin.4de0a22a/qt.headers -I/usr/include/qt5/QtCore/5.11.2 -I/usr/include/qt5/QtCore/5.11.2/QtCore -I/usr/include/qt5/QtDBus -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtGui/5.11.2 -I/usr/include/qt5/QtGui/5.11.2/QtGui -I/usr/include/qt5/QtServiceSupport -I/usr/include/qt5/QtServiceSupport/5.11.2 -I/usr/include/qt5/QtServiceSupport/5.11.2/QtServiceSupport -I/usr/include/qt5/QtWaylandClient -I/usr/include/qt5/QtWaylandClient/5.11.2 -I/usr/include/qt5/QtWaylandClient/5.11.2/QtWaylandClient -std=c++11 -o /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/build/default/materialdecorationplugin.4de0a22a/3a52ce780950d4d9/plugin.cpp.o -c /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/src/plugins/decorations/material/plugin.cpp

ERROR: In file included from /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/src/plugins/decorations/material/materialdecoration.h:43,
                 from /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/src/plugins/decorations/material/plugin.cpp:24:
/usr/include/qt5/QtWaylandClient/5.11.2/QtWaylandClient/private/qwaylandshellsurface_p.h:59:10: fatal error: QtWaylandClient/private/qwayland-wayland.h: No such file or directory
 #include <QtWaylandClient/private/qwayland-wayland.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
ERROR: /usr/bin/g++ -g -O0 -Wall -Wextra -m32 -pipe -fexceptions -fvisibility=hidden -fvisibility-inlines-hidden -fPIC -DQT_CORE_LIB -DQT_DBUS_LIB -DQT_GUI_LIB -DQT_SERVICE_SUPPORT_LIB -DQT_WAYLANDCLIENT_LIB -I/builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/src/plugins/decorations/material -I/usr/include/qt5 -I/usr/include/qt5/QtCore -I/usr/lib/qt5/mkspecs/linux-g++ -I/builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/build/default/materialdecorationplugin.4de0a22a/qt.headers -I/usr/include/qt5/QtCore/5.11.2 -I/usr/include/qt5/QtCore/5.11.2/QtCore -I/usr/include/qt5/QtDBus -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtGui/5.11.2 -I/usr/include/qt5/QtGui/5.11.2/QtGui -I/usr/include/qt5/QtServiceSupport -I/usr/include/qt5/QtServiceSupport/5.11.2 -I/usr/include/qt5/QtServiceSupport/5.11.2/QtServiceSupport -I/usr/include/qt5/QtWaylandClient -I/usr/include/qt5/QtWaylandClient/5.11.2 -I/usr/include/qt5/QtWaylandClient/5.11.2/QtWaylandClient -std=c++11 -o /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/build/default/materialdecorationplugin.4de0a22a/73489b871e127538/moc_materialdecoration.cpp.o -c /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/build/default/materialdecorationplugin.4de0a22a/qt.headers/moc_materialdecoration.cpp

ERROR: In file included from /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/build/default/materialdecorationplugin.4de0a22a/qt.headers/../../../../src/plugins/decorations/material/materialdecoration.h:43,
                 from /builddir/build/BUILD/materialdecoration-5ad9d89f613ac795875eb142b89f6c022d7412cd/build/default/materialdecorationplugin.4de0a22a/qt.headers/moc_materialdecoration.cpp:9:
/usr/include/qt5/QtWaylandClient/5.11.2/QtWaylandClient/private/qwaylandshellsurface_p.h:59:10: fatal error: QtWaylandClient/private/qwayland-wayland.h: No such file or directory
 #include <QtWaylandClient/private/qwayland-wayland.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
ERROR: Process failed with exit code 1.
The following products could not be built for configuration default:
materialdecorationplugin
```

**Comment by plfiorini** [comment]
Upstream bug: https://bugreports.qt.io/browse/QTBUG-71340

**Comment by plfiorini** [comment]
Upstream problem was fixed since Qt 5.12.
We have 5.12, 5.14 and 5.15 in our CI and all of them work.
